sale_layout_category_hide_detail/views/account_portal_templates.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="portal_invoice_report_layouted_hide_detail" inherit_id="account.portal_invoice_report">
+        <xpath expr="//span[@t-field='l.name']/../../.." position="replace">
```

Comment:
Please NEVER do a replace. You should hide elements you don't want, or modify other attributes, but this is a total enemy of compatibility between several modules. Imagine there's other module that adds an extra column to each line. With this, you are removing the possibility of showing it.